platform_channels/test/src/pet_list_screen_test.dart

```diff
@@ -22,12 +22,12 @@ void main() {
       }
     ];
 
-    PetListModel petListModel;
+    PetListModel? petListModel;
 
     setUpAll(() {
       // Mock for the pet list received from the platform.
       basicMessageChannel.setMockMessageHandler((message) async {
-        petListModel = PetListModel.fromJson(message);
+        petListModel = PetListModel.fromJson(message!);
```

Comment:
If possible, I'd like to eliminate the need for `!` all over the place.

Reply:
Sure @domesticmouse!